ChaosMod/Effects/db/Vehs/VehsDetachWheel.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+Effect by Reguas
+*/
+
+#include "Memory/EntityFragments.h"
+#include <stdafx.h>
```

Comment:
Should be the first include, add an empty line between both to make sure they don't get reordered by the formatter

```suggestion
#include <stdafx.h>

#include "Memory/EntityFragments.h"
```